README.md

```diff
@@ -1 +1,2 @@
-# code-katas
\ No newline at end of file
+# code-katas
+This repo holds th code to my code katas on codewars.com
```

Comment:
That's a start. What's some of the code you've written though? How does a newcomer to your repo know what they're looking at before they look at it?